neuromation/cli/job.py

```diff
@@ -436,16 +425,12 @@ async def exec(
     neuro exec --no-tty my-job ls -l
     """
     real_cmd = _parse_cmd(cmd)
-    id = await resolve_job(
+    job = await resolve_job(
         job, client=root.client, status={JobStatus.PENDING, JobStatus.RUNNING}
     )
-    if interactive is None:
-        interactive = tty
-    async with async_timeout.timeout(timeout if timeout else None,):
-        retcode = await root.client.jobs.exec(
-            id, shlex.split(real_cmd), tty=tty, no_key_check=no_key_check,
-        )
-        sys.exit(retcode)
+    _check_tty(root, tty)
+    async with async_timeout.timeout(timeout):
+        await process_exec(root, job, real_cmd, tty)
```

Comment:
`no_key_check` is ignored now.

Reply:
Yes. It was used for tests only.
I'll keep it for a while. Just in case.